wqflask/wqflask/correlation/pre_computes.py

```diff
@@ -156,3 +160,39 @@ def get_datasets_data(base_dataset, target_dataset_data):
         samples_fetched, base_traits_data)
 
     return (target_results, base_results)
+
+
+def get_file_size(file_path):
```

Comment:
*issue:* AFAICT, this function is only used once.  No need defining functions that are only used once.  Also, we just checking the file size, which can be done in 1 line:

```
import os
os.stat("<file/patch>").st_size / (1024 ** 2)  # in Mb
```